deployment/ansible/deploy.yml

```diff
@@ -166,6 +166,14 @@
         when: genesis_accounts is defined
         tags: [config]
 
+       - name: "Copy node keys to {{ project_root }}/{{ config.keypair.dir }}"
```

Comment:
Could you please update the title as well, it's the same as the chain keys

`"Copy node peer keys to {{ project_root }}/{{ config.keypair.dir }}"` should do